src/circuit.rs

```diff
@@ -1,6 +1,7 @@
 use std::fmt::{self, Display};
 
 use crate::gate::Gate;
+use crate::prelude::One;
```

Comment:
Use `crate::prelude` only outside the main library. Here it is meant only to export (ok, in Rust there is no problem of being circular, but better to take things from the source).

Reply:
Thanks, fixed in 77ab54ffda48b2c35db6b428f05960f4b9aa9bd9. I think it was the compiler's suggestion.